
char command
---
float32 intial_velocity
